config/routes.rb

```diff
@@ -2,6 +2,7 @@
   devise_for :users
   root to: 'pages#home'
 
+  get "items/yourlistings", to: "items#listed", as: 'items_listed' #search for user's own listings
```

Comment:
can we route to get users/:user_id/items instead. since the listings belong to a specific user so may make more sense to route into a nested route